buildscripts/travis-build.sh

```diff
@@ -14,10 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+MASTER="master"
+if [ "$TRAVIS_BRANCH" != "" ]; 
+then
+ if [ "$TRAVIS_BRANCH" == "$MASTER" ];
+ then
+  IMAGE_TAG="ci"
+ else
+  IMAGE_TAG="$TRAVIS_BRANCH-ci"
+ fi
+else
+ IMAGE_TAG="local"
```

Comment:
Having just the tag as local does not seem right? It might be ok if used along with org name i.e. contributor/m-apiserver:local vs openebs/m-apiserver:local

Reply:
@AmitKumarDas We are here making images like, "openebs/m-apiserver:local" for local developer level building(to avoid TRAVIS_BRANCH env not set). Do you mean to have contributor/m-apiserver:local? 
Alternate Suggestion: Or shall we have openebs/m-apiserver: contributors-branch-name (obtained from git branch). Please confirm

Reply:
openebs/m-apiserver:local of mine will override yours while we are doing the code, testing & pushing this image of our respective features in parallel. Hence I felt the openebs org should not be used at all. It can be individual's org.

Reply:
the images built with local tag will be built only on developer's personal machine(if travis-branch is not set, handled in above code). Our push script will run only in travis. We cannot get developer's docker repo(for pushing), This will not be used on travis. When we make a pr, while doing `make all` to verify everything is working fine, we are trying to add a temporary tag, only specific to local machine.

Reply:
will comment

Reply:
looks fine